spec/puppet-lint_spec.rb

```diff
@@ -4,23 +4,25 @@
   subject { PuppetLint.new }
 
   it 'should accept manifests as a string' do
-    subject.code = "class foo { }"
+    subject.code = 'class foo { }'
     expect(subject.code).to_not be_nil
   end
 
+  # rubocop:disable Style/FormatString
```

Comment:
Are these two tests necessary? They appear to be testing ruby functionality rather than puppet-lint's.

Reply:
In place due to the monkey patches to enable `String#%` on very old versions of ruby. Those monkey patches definitely need to go away, but that's out of the scope of this PR which is just rubocop related changes.